packages/dev/core/src/Physics/v1/Plugins/cannonJSPlugin.ts

```diff
@@ -313,7 +313,7 @@ export class CannonJSPlugin implements IPhysicsEnginePlugin {
     }
 
     private _checkWithEpsilon(value: number): number {
-        return value < PhysicsEngine.Epsilon ? PhysicsEngine.Epsilon : value;
+        return value < PhysicsEngineV1.Epsilon ? PhysicsEngineV1.Epsilon : value;

```

Comment:
This epsilon can move to the Constants file and be taken out of the physics engine altogether (while maintaining back-compat referencing it from the physics engine)

Reply:
Fixed